lib/bots/versionbot.ts

```diff
@@ -538,13 +540,11 @@ export class VersionBot extends ProcBot {
             // Ensure that all the statuses required have passed.
             // If not, an error will be thrown and not proceed any further.
             return this.checkStatuses(prInfo, ghApiCalls);
-        }).then((statusesPassed) => {
+        }).then((checkStatus) => {
             // Finally we have an array of booleans. If any of them are false,
             // statuses aren't valid.
-            if (!statusesPassed) {
-                throw new Error(`At least one status check has failed; ${process.env.VERSIONBOT_NAME} will not ` +
-                    'proceed to update this PR unless forced by re-applying the ' +
-                    `\`${MergeLabel}\` label`);
+            if ((checkStatus === StatusChecks.Failed) || (checkStatus === StatusChecks.Pending)) {
+                throw new Error('checksPendingOrFailed');
```

Comment:
Yep, it's a better way of doing it. I've got that under some architectural changes I want to do in a big lump anyway. There's a few of these around, so I'd like to change them all at the same time.